src/openrct2/network/Network.cpp

```diff
@@ -3657,7 +3665,8 @@ void game_command_kick_player(
     NetworkPlayer* kicker = gNetwork.GetPlayerByID(game_command_playerid);
     if (player == nullptr)
     {
-        // Player might be already removed by the PLAYERLIST command, need to refactor non-game commands executing too early.
+        // Player might be already removed by the PLAYERLIST command, need to refactor non-game commands executing too
+        // early.
```

Comment:
Not sure how this happened. I suspect clang-format, but it doesn't happen again when I try to trigger it.